Zukini/Hooks.cs

```diff
@@ -55,8 +55,7 @@ protected FeatureContext FeatureContext
         protected void BeforeScenario()
         {
             // Create a property bucket so we have a place to store values between steps
-            var propertyBucket = new PropertyBucket();
-            _objectContainer.RegisterInstanceAs<PropertyBucket>(propertyBucket);
+            var propertyBucket = _objectContainer.Resolve<PropertyBucket>();
```

Comment:
We found that the Resolve() automatically registers if the object has a parameter-less constructor and is not registered yet. This is why the exception was being thrown, the BeforeScenario in the project is being called prior to the BeforeScenario in the base.